doc/zmq.md

```diff
@@ -33,8 +33,10 @@ buffering or reassembly.
 
 ## Prerequisites
 
-The ZeroMQ feature in Bitcoin Core requires ZeroMQ API version 4.x or
-newer. Typically, it is packaged by distributions as something like
+The ZeroMQ feature in Bitcoin Core requires the ZeroMQ API
```

Comment:
Why get rid of version limit?

Reply:
it's already centrally located in doc/dependencies.md. this removes redundant info that can get out of synch later.